src/Microsoft.TestPlatform.ObjectModel/Navigation/NativeMethods.cs

```diff
@@ -0,0 +1,651 @@
+
+        int Stub82();
+
+        [PreserveSig]
+        int FindChildren(SymTagEnum tag, string str, int flags, out IDiaEnumSymbols symbol);
+
+        int Stub84();
+
+        int Stub85();
+
+        int Stub86();
+
+        int Stub87();
+
+        int Stub88();
+
+        int Stub89();
+
+        int Stub90();
+
+        int Stub91();
+
+        int Stub92();
+
+        int Stub93();
+
+        int Stub94();
+
+        int Stub95();
+
+        int Stub96();
+
+        int Stub97();
+
+        int Stub98();
+
+        int Stub99();
+
+        int Stub100();
+
+        int Stub101();
+
+        int Stub102();
+
+        int Stub103();
+
+        int Stub104();
+
+        int Stub105();
+
+        int Stub106();
+
+        int Stub107();
+
+        int Stub108();
+
+        int Stub109();
+
+        int Stub110();
+
+        int Stub111();
+
+        int Stub112();
+
+        int Stub113();
+
+        int Stub114();
+
+        int Stub115();
+
+        int Stub116();
+
+        int Stub117();
+
+        int Stub118();
+
+        int Stub119();
+
+        int Stub120();
+
+        int Stub121();
+
+        int Stub122();
+
+        int Stub123();
+
+        int Stub124();
+
+        int Stub125();
+
+        int Stub126();
+
+        int Stub127();
+
+        int Stub128();
+
+        int Stub129();
+
+        int Stub130();
+
+        int Stub131();
+
+        int Stub132();
+
+        int Stub133();
+
+        int Stub134();
+
+        int Stub135();
+
+        int Stub136();
+
+        int Stub137();
+
+        int Stub138();
+
+        int Stub139();
+
+        int Stub140();
+
+        int Stub141();
+
+        int Stub142();
+
+        int Stub143();
+
+        int Stub144();
+
+        int Stub145();
+
+        int Stub146();
+
+        int Stub147();
+
+        int Stub148();
+
+        int Stub149();
+
+        int Stub150();
+
+        int Stub151();
+
+        int Stub152();
+
+        int Stub153();
+
+        int Stub154();
+
+        int Stub155();
+    }
+
+    // The definition for DiaSource COM object is present InternalApis\vctools\inc\dia2.h
+    // The GUID here must match what is present in dia2.h
+    [ComImport, CoClass(typeof(DiaSourceClass)), Guid("79F1BB5F-B66E-48E5-B6A9-1545C323CA3D")]
+    internal interface DiaSource : IDiaDataSource
+    {
+    }
+
+    // The definition for DiaSourceClass COM object is present InternalApis\vctools\inc\dia2.h
+    // The GUID here must match what is present in dia2.h
+    [ComImport, ClassInterface((short)0), Guid("E6756135-1E65-4D17-8576-610761398C3C")]
+    internal class DiaSourceClass
+    {
+    }
+
+    internal static class DiaSourceObject
+    {
+        [DllImport("kernel32.dll", SetLastError = true)]
+        public static extern IntPtr LoadLibraryEx(string lpFileName, IntPtr hReservedNull, int dwFlags);
+
+        public static IDiaDataSource GetDiaSourceObject()
+        {
+            var currentDirectory = new ProcessHelper().GetCurrentProcessLocation();
```

Comment:
Check how we are handling `/Inproc` scenario.

Reply:
Will check this once InProc comes into play. We will have to make some changes then definitely, will let faizan know